### DISCUSSION TITLE
duplicate jobs on saving twice

### DISCUSSION THREAD

**Discussion by neerajmalve** [alseambusher/crontab-ui#172] [General]
This is a very nice utility and I would like to use it for a production environment, a couple of issues I found here though.
1. When I installed on a host which had few crontab entries, It didn't show/detect any while fetching. (If the default crontab format was not readable for this package, at least it should warn something like -  3 Cron Entries found, but unable to Fetch them)
2. When no entries are fetched, and I create 1 test entry, those un-detected entries were gone too. If not able to detect, they should be left out as it is?
3. This is a very important issue and needs a fix - when I create entry thru crontab-UI and fetch it back. Or save it twice. It makes a duplicate entry.. which sounds like a bug.

**Comment by KingsleyChimezie** [comment]
Can we get this created as an issue. It's present for me in v0.3.16.
